frontend/pages/volunteer.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from "next";
+import { IVolunteer } from "types/volunteer";
+import Head from "next/head";
+import { Volunteer, VolunteersQuery, VolunteersDocument, FaqsQuery, FaqsDocument, Faq } from "../graphql/generated";
```

Comment:
Really nice how you've managed to integrate all the generated queries and documents, definitely saves on a lot of custom code and should provide much stronger type-definitions to help us avoid mistakes 🥳